app/src/main/java/com/example/rent/application1/models/Picture.java

```diff
@@ -0,0 +1,96 @@
+package com.example.rent.application1.models;
+
+import android.media.Image;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Picture implements Parcelable{
+
+    private String filename;
+    private String author;
+    private int height;
+    private int width;
+    private int id;
+
+    protected Picture(Parcel in) {
+        filename = in.readString();
+        author = in.readString();
+        height = in.readInt();
+        width = in.readInt();
+        id = in.readInt();
+    }
+
+    public static final Creator<Picture> CREATOR = new Creator<Picture>() {
+        @Override
+        public Picture createFromParcel(Parcel in) {
+            return new Picture(in);
+        }
+
+        @Override
+        public Picture[] newArray(int size) {
+            return new Picture[size];
+        }
+    };
+
+    public int getHeight() {
+        return height;
+    }
+
+    public void setHeight(int height) {
+        this.height = height;
+    }
+
+    public int getWidth() {
+        return width;
+    }
+
+    public void setWidth(int width) {
+        this.width = width;
+    }
+
+    public static String getThumbnailUrl(Picture picture) {
+        return "https://unsplash.it/300/200/?image=" + picture.getId();
```

Comment:
Wartości `300` i `200` wyrzuć do stałych

Reply:
I łącz stringi przy pomocy `String.format()`

Reply:
Rozumiem idee łączenia stringów przy pomocy String.format(), ale na chwilę obecną jest dla mnie to dużo mniej czytelne niż normalne łączenie Stringów.